src/test/EigenPod.t.sol

```diff
@@ -1256,29 +1256,6 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
         Merkle.verifyInclusionSha256(proof, root, leaf, index);
     }
 
-    /// @notice Test that the Merkle proof verification fails when the proof length is empty
-    function testVerifyInclusionKeccakFailsForEmptyProof(bytes32 root, bytes32 leaf, uint256 index) public {
-        bytes memory emptyProof = new bytes(0);
-        cheats.expectRevert(
-            bytes("Merkle.processInclusionProofKeccak: proof length should be a non-zero multiple of 32")
-        );
-        Merkle.verifyInclusionKeccak(emptyProof, root, leaf, index);
-    }
-
-    /// @notice Test that the Merkle proof verification fails when the proof length is not a multiple of 32
-    function testVerifyInclusionKeccakFailsForNonMultipleOf32ProofLength(
-        bytes32 root,
-        bytes32 leaf,
-        uint256 index,
-        bytes memory proof
-    ) public {
-        cheats.assume(proof.length % 32 != 0);
-        cheats.expectRevert(
-            bytes("Merkle.processInclusionProofKeccak: proof length should be a non-zero multiple of 32")
-        );
-        Merkle.verifyInclusionKeccak(proof, root, leaf, index);
-    }
```

Comment:
Hmm... this should be fine to remove. I checked `BeaconChainProofs`, and we're already requiring specific lengths for each of the proofs, so we shouldn't need extra checks/tests here.

Reply:
oh yeah and we're using sha256 there, anyway!

Reply:
Yup, I checked that the Merkle keccak256 functions weren't being used anywhere